state/upgrades.go

```diff
@@ -220,3 +221,20 @@ func stripLocalFromFields(st *State, collName string, fields ...string) ([]txn.O
 	}
 	return ops, nil
 }
+
+func DropOldLogIndex(st *State) error {
+	// If the log collection still has the old e,t index, remove it.
+	key := []string{"e", "t"}
+	db := st.MongoSession().DB(logsDB)
+	collection := db.C(logsC)
+	err := collection.DropIndex(key...)
+	if err == nil {
+		return nil
+	}
+	if queryErr, ok := err.(*mgo.QueryError); ok {
```

Comment:
I was thinking you could check the output of Indexes() and then decide whether to drop, but this is fine too.